__tests__/app.test.js

```diff
@@ -40,12 +40,71 @@ describe("3- GET /api/topics", () => {
       });
   });
 
-  test("404 - bad request response for an invalid path - returns error message", () => {
+  test("404 - handles bad path", () => {
     return request(app)
-      .get("/api/topicz")
+      .get("/api/invalid_path")
       .expect(404)
       .then(({ body: { msg } }) => {
-        expect(msg).toBe("Error - No such path");
+        expect(msg).toBe("bad path");
+      });
+  });
+});
+
+describe("04 GET /api/articles/:article_id", () => {
+  test("200 - returns article object with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toBeInstanceOf(Object);
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          })
```

Comment:
can we have a more specific test to ensure that the correct article_id is being sent back instead of just any number please

Reply:
thanks Heather, will amend and resend request